pkgs/misc/vim-plugins/generated.nix

```diff
@@ -1713,6 +1713,26 @@ final: prev:
     meta.homepage = "https://github.com/rktjmp/fwatch.nvim/";
   };
 
+  fzf-checkout-vim = buildVimPluginFrom2Nix {
+    name = "fzf-checkout.vim";
+    version = "2021-07-21";
+
+    src = fetchFromGitHub {
+      owner = "stsewd";
+      repo = "fzf-checkout.vim";
+      rev = "4d5ecae74460de8fed4f743f6bd53c4c31d32797";
+      sha256 = "0mia7p2z8l3lrid0v8ml4i8y190gh4ll9898yyg4gcghhxp83zpm";
+    };
+
+    # The plugin has a makefile which tries to run tests in a docker container.
+    # This prevents it.
+    prePatch = ''
+      rm Makefile
+    '';
+
+    meta.homepage = "https://github.com/stsewd/fzf-checkout.vim";
+  };
+
```

Comment:
That does not work and will be overwritten when regenerated. Please move the extra things to https://github.com/NixOS/nixpkgs/blob/master/pkgs/misc/vim-plugins/overrides.nix and generate the file.